packer/install-dependencies.sh

```diff
@@ -18,6 +20,7 @@ cp $SYSCTL_CONFIG_SRC $SYSCTL_CONFIG_DEST
 sysctl_reload 20 5 10
 cp $ETC_ISSUE_CONFIG_SRC $ETC_ISSUE_CONFIG_DEST
 cp $ETC_ISSUE_NET_CONFIG_SRC $ETC_ISSUE_NET_CONFIG_DEST
+cp $MODPROBE_CIS_SRC $MODPROBE_CIS_DEST
```

Comment:
should this be before the reload?

Reply:
As far as my testing uncovered, `sysctl --system` was not necessary to pull in updated kernel module config.

Reply:
Honestly I think there's a problem here, the `sysctl_reload` call doesn't *really* need to be called vi VHD CI (it'll be applied when it boots up), but if we are calling it we should at least error out if it fails as a kind of basic validation.